Code/GraphMol/MolInterchange/Writer.cpp

```diff
@@ -38,6 +39,8 @@ namespace RDKit {
 namespace MolInterchange {
 
 namespace {
+constexpr int MAX_DECIMAL_PLACES = 4;
```

Comment:
Note for us for later: we should modify the mol serialization code so that it includes more than 4 decimal points on atomic positions

but that's not for this PR